default_app/default_app.ts

```diff
@@ -44,7 +44,7 @@ ipcMain.handle('bootstrap', (event) => {
 async function createWindow () {
   await app.whenReady();
 
-  const options: Electron.BrowserWindowConstructorOptions = {
+  const options: Electron.BrowserWindowOptions = {
```

Comment:
Changing from `Electron.BrowserWindowConstructorOptions` to `Electron.BrowserWindowOptions` is a breaking change for folks who use TypeScript. Can you please document this in `docs/breaking-changes.md` ?